Dockerfile.aarch64

```diff
@@ -0,0 +1,83 @@
+###
+# Build stage
+##
+FROM balenalib/raspberrypi3-node:10-build as build
```

Comment:
According to https://www.balena.io/docs/reference/base-images/devicetypes/ `raspberrypi3` is `armv7hf`, not `aarch64` ?

Reply:
yes, we use the rpi3 base image for 32-bit support

Reply:
So why is the rpi3 base image being used in `Dockerfile.aarch64`? Apologies if I'm not understanding something.

Reply:
@lurch because the 32-bit raspberrypi base image also has the raspbian repositories, so it's not directly the `armv7hf` base image. Here those might or might not be used, but for other device types, e.g. a Pi3, it *would* be a `balenalib/raspberrypi3-node` base image, and it's better to be on par, than one device type using the architecture base image, the other the device type bsae image (where they are not direcly the same)

Reply:
@rahul-thakoor maybe I would specify `10-buster-build` here and in the other one, to ensure it's the right kind of debian, and less surprising.

Also, have you tried to build with `12-`? If we could bump that it would be grand!

Reply:
I still don't quite see why the "32-bit raspberrypi base image" is being used in the 64-bit Dockerfile.aarch64, but I'm not really that bothered so I won't bother hassling you any further :slightly_smiling_face:

Reply:
@lurch there might be cases where 64bit packages or libraries are not available. so using 32-bit base image we can still make use of those on a 64bit system. Perhaps the following 2 PRs might be helpful to understand:
1. https://github.com/balena-io-playground/balena-senseoflife/pull/2/files
2. https://github.com/balena-io-playground/balena-senseoflife/pull/3

Reply:
Oh, so even though the filename says Dockerfile.aarch64, the actual built image is actually all 32-bit code?

Reply:
Yes, `Dockerfile.aarch64` targets 64bit systems and on those, we explicitly use 32-bit images otherwise they would default to the 64-bit ones